.github/workflows/checks.yml

```diff
@@ -35,9 +35,12 @@ jobs:
       run: mypy --python-version ${{ matrix.python }} -p build
 
 
-  pre-commit: # per https://github.com/pre-commit/action
+  pre-commit:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - uses: actions/setup-python@v1
-    - uses: pre-commit/action@v2.0.0
+    - name: Checkout
+      uses: actions/checkout@v2
```

Comment:
Missing the new lines between steps.

Reply:
https://github.com/FFY00/python-build/issues/51 to address this